contracts/seed/Seed.sol

```diff
@@ -394,6 +398,26 @@ contract Seed {
         emit TokensClaimed(_funder, _claimAmount);
     }
 
+    function claimTip() external returns (uint256) {
+        require(
+            endTime < block.timestamp || maximumReached,
+            "Seed: the distribution has not yet finished"
+        );
```

Comment:
Could you point me to the BL logic, that specifies this?

Reply:
I don't think any BL for the claiming of tip has been written down in the BL section. We have, however, decided that the tip should have its own cliff and duration.
I have based this requirement on the fact that the cliff and duration count from the point the Seed has been completed.

Reply:
Can you then reach out to Blink, and make sure, that it is documented?